app/src/main/res/menu/feeds_apply_action_options.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+      xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <item
+        android:id="@+id/sort"
+        android:icon="?attr/ic_sort"
+        android:title="@string/sort"
+        app:showAsAction="always">
+        <menu>
+            <item android:id="@+id/sort_title_a_z"
+                  android:title="@string/sort_title_a_z"/>
+            <item android:id="@+id/sort_title_z_a"
+                  android:title="@string/sort_title_z_a"/>
+<!--            <item android:id="@+id/sort_date_new_old"-->
+<!--                  android:title="@string/sort_date_new_old"/>-->
+<!--            <item android:id="@+id/sort_date_old_new"-->
+<!--                  android:title="@string/sort_date_old_new"/>-->
+        </menu>
+    </item>
+
+    <item
+        android:id="@+id/select_options"
+        android:icon="?attr/ic_filter"
+        android:title="@string/filter"
+        app:showAsAction="always">
+
+        <menu>
+            <item android:id="@+id/check_all"
+                android:title="@string/all_label"/>
+            <item android:id="@+id/check_none"
+                  android:title="@string/select_none_label"/>
+            <item android:id="@+id/check_counter_gt_zero"
+                android:title="@string/subscriptions_counter_greater_zero"/>
+            <item android:id="@+id/check_autodownload_enabled"
+                android:title="@string/auto_downloaded"/>
+            <item android:id="@+id/check_autodownload_not_enabled"
+                android:title="@string/not_auto_downloaded"/>
```

Comment:
That's what I meant when I said that it might be easier to have this on the existing subscriptions screen. All these filters now need to be re-implemented even though they already exist on the normal screen. Maybe they could be removed here. Then it could just take the filter from the normal subscriptions screen and users just need to apply filters before opening the multi-select screen.